extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -194,7 +194,18 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 							continue;
 						}
 						try {
-							connectionString = await vscodeMssqlApi.getConnectionString(connectionUri, false, false);
+							// Prompt to include password in connection string
+							let includePassword = await vscode.window.showQuickPick([constants.yesString, constants.noString], {
+								title: constants.includePassword,
+								canPickMany: false,
+								ignoreFocusOut: true
+							});
+							if (includePassword === constants.yesString) {
+								connectionString = await vscodeMssqlApi.getConnectionString(connectionUri, true, false);
+							} else {
+								connectionString = await vscodeMssqlApi.getConnectionString(connectionUri, false, false);
+								await vscode.window.showInformationMessage(constants.userPasswordLater);
```

Comment:
This doesn't need to be blocking I think.

Also we should provide a button to open up the file when clicked.